impl/src/main/java/org/ehcache/impl/config/store/disk/OffHeapDiskStoreConfiguration.java

```diff
@@ -61,6 +80,15 @@ public int getWriterConcurrency() {
   }
 
   /**
+   * Returns the number of disk segments created
+   *
+   * @return the number of segments
+   */
+  public int getDiskConcurrency() {
```

Comment:
Yes. I was using diskSegments at first. Then thought it was too low level and used the parameter name to `EhcachePersistentConcurrentOffHeapClockCache` which is concurrency. But wasn't quite happy either.

So I can go back to getDiskSegments.

I will add the tests and the documentation. Then we can wait for 3.4.